modules/openapi-generator/src/main/resources/r/api.mustache

```diff
@@ -93,13 +93,13 @@
       {{#isApiKey}}
```

Comment:
in the lines above at line 91, the usage of jsonlite::base64_enc() is not working when the authentication string is longer.
The issue is that the above method  is encoding but adding a extra '\n' in the output of the encoded string.

Please refer to a similar issue in the below link:
[jsonline issue](https://github.com/jeroen/jsonlite/issues/220)

Please make a fix or any workaround. Thanks in advance

Reply:
Thanks. Will use other base64 encoding function instead. Can you please open an issue to track this instead?

Reply:
@Ramanth I've filed https://github.com/OpenAPITools/openapi-generator/pull/2171 to fix the issue. Can you please check to see if that works for you?

Reply:
#2171 merged into r_refactor. Please give it a try.